packages/venia-ui/lib/components/WishlistPage/wishlistMoreActionsDialog.css

```diff
@@ -0,0 +1,20 @@
+.root {
+    display: grid;
+    grid-auto-flow: row;
+}
+
+.row {
+    align-items: center;
+    display: grid;
+    gap: 2rem;
+    grid-template-columns: auto 1fr auto;
+    padding: 1rem;
+    text-align: left;
+}
+
+.rowAlternate {
+    composes: row;
+
+    border-top: 2px solid rgb(var(--venia-global-color-gray));
+    border-bottom: 2px solid rgb(var(--venia-global-color-gray));
+}
```

Comment:
I'm not opposed to an `nth-child(even)` pseudo selector here.

Reply:
https://github.com/magento/pwa-studio/pull/2793/commits/565c6d36dd1ea7e04f19ba90d179fc08084fecc2